multi-node/aws/pkg/cluster/stack_template.go

```diff
@@ -589,6 +596,48 @@ func StackTemplateBody(defaultArtifactURL string) (string, error) {
 		"Description": "Specific availability zone (optional)",
 	}
 
+	par[parVpcCIDR] = map[string]interface{}{
```

Comment:
I'm a little confused. Why are you adding all of these template parameters if only parVpcCIDR, parInstanceCIDR and parControllerIP are being used? The rest just need to get rendered into the cloud-config, but not be part of the CloudFormation Template, right?